src/types/post.ts

```diff
@@ -0,0 +1,15 @@
+import { IsBoolean, IsNotEmpty, IsOptional, IsString } from 'class-validator'
+
+export class PostSubmitDto {
```

Comment:
可能 `CreatePostDto` 会更容易理解
```suggestion
export class CreatePostDto {
```

Reply:
因为update也会用到这个dto，CreatePostDto的话，因为Update也用，就觉得会有语义不明的情况